.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "${workspaceFolder}/ENV/bin/python"
+}
```

Comment:
Delete this file, it is unnecessary. Then add it to the `.gitignore` to avoid it popping up again.